### PR TITLE
Bump tektoncd/pipeline and tektoncd/triggers

### LINKED ISSUE #645: Bump pipeline dependency to 0.10.0
Let's bump the pipeline dependency to latest 0.10.0

### LINKED ISSUE #646: Bump triggers dependency to v0.2.0
Triggers has been updated to 0.2.0 let's bump it

/assign @piyush-garg

### PR DESCRIPTION
**Pull request by piyush-garg**
This will bump the tektoncd/pipeline dep to 0.10.1
and bump tektoncd/triggers upto the commit
https://github.com/tektoncd/triggers/tree/68f57133472285df9990804bee124ae3da2ac4df
as we need triggers with bumped pipeline dep
to 0.10.1

Fix #645
Fix #646

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [x] Run the code checkers with `make check`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
Bump tektoncd-pipeline to 0.10.1
and tektoncd/triggers to 0.2.1
```

### CODE REVIEW

**Comment by danielhelfand** [comment]
Oh wait, are you doing triggers as well here?

**Comment by piyush-garg** [comment]
@danielhelfand Yaa I bumped both pipeline and triggers as bumping one is not working, we have a dependency because of resourceclient.

As the triggers PR is merged, this PR is ready for review.

cc @vdemeester @chmouel

File: go.mod [hunk 1]
```diff
@@ -3,64 +3,53 @@ module github.com/tektoncd/cli
...
 	github.com/pkg/errors v0.8.1
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.4.0 // indirect
-	github.com/tektoncd/pipeline v0.9.2
+	github.com/tektoncd/pipeline v0.10.1
 	github.com/tektoncd/plumbing v0.0.0-20191218171343-56a836c50336
-	github.com/tektoncd/triggers v0.1.1-0.20191218175743-c5b8b4d9ee00
-	go.opencensus.io v0.22.1 // indirect
+	github.com/tektoncd/triggers v0.2.1-preview.0.20200130181252-68f571334722
```
Comment (danielhelfand) [inline go.mod]: Should we wait for official release?

Comment (piyush-garg) [reply go.mod]: Even after the official release, it is the same. No effects

**Pull request merged**